dashboards-reports/public/components/report_definitions/utils/utils.tsx

```diff
@@ -99,34 +100,17 @@ export const definitionInputValidation = async (
       error = true;
     }
   }
-  // if email delivery
-  if (metadata.delivery.delivery_type === 'Channel') {
-    // no recipients are listed
-    if (metadata.delivery.delivery_params.recipients.length === 0) {
-      setShowEmailRecipientsError(true);
-      setEmailRecipientsErrorMessage(
-        i18n.translate(
-          'opensearch.reports.error.emailRecipientsListCannotBeEmpty',
-          { defaultMessage: 'Email recipients list cannot be empty.' }
-        )
-      );
-      error = true;
-    }
-    // recipients have invalid email addresses: regexp checks format xxxxx@yyyy.zzz
-    let emailRegExp = /\S+@\S+\.\S+/;
-    let index;
-    let recipients = metadata.delivery.delivery_params.recipients;
-    for (index = 0; index < recipients.length; ++index) {
-      if (recipients[0].search(emailRegExp) === -1) {
-        setShowEmailRecipientsError(true);
-        setEmailRecipientsErrorMessage(
-          i18n.translate('opensearch.reports.error.invalidEmailAddresses', {
```

Comment:
also need to remove related translation from pl.json?

Reply:
Will remove